src/drivers/cmakeDriver.ts

```diff
@@ -276,6 +276,8 @@ export abstract class CMakeDriver implements vscode.Disposable {
         let envs;
         if (this.useCMakePresets) {
             envs = EnvironmentUtils.create(configurePreset ? configurePreset.environment : this._configurePreset?.environment);
+            envs = EnvironmentUtils.merge([envs, await this.computeExpandedEnvironment(this.config.environment, envs)]);
```

Comment:
Does this mean we'll (perhaps unintentionally) support preset style ${} macros in the settings.json specified environment?